src/pspm_convert_pixel2unit.m

```diff
@@ -161,7 +161,13 @@
 
 if strcmpi(unit,'degree')
     options.channel_action = 'replace';
-    [lsts, outinfo] = pspm_compute_visual_angle(fn,0,width, height, distance,unit_h_w_d,options);
+    [~, lsts, outinfo] = evalc(['pspm_compute_visual_angle(fn,outinfo.channel,'...
```

Comment:
I agree with passing `outinfo.channel` as a parameter, but I don't think we need to use `evalc`. Why not just call the function directly?

Reply:
Yeah sorry it was my previous solution and I've forgotten to change it.